phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerWineToolsTab.java

```diff
@@ -51,6 +51,10 @@ private void populate(ApplicationDTO engineTools) {
         final VBox toolsPane = new VBox();
         final Text title = new TextWithStyle(tr("Wine tools"), TITLE_CSS_CLASS);
 
+        if (engineTools == null) {
+            return;
```

Comment:
Just asking:
Should this happen, that null is passed?
If this is the case I think we should think about passing an `Optional` object to the method.

Reply:
It depends on the repository really. It might be that you do not have engine tools in your scripts. I'm currently not quite sure which is the best place to check this. So at the moment, this is only a safety guard to avoid a crash in case this happens.

Reply:
Then I think we should add a `TODO: check later if null should really happen`.

Reply:
I will implement a solution.